project/src/main/java/com/google/sps/classes/Cafeteria.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps.classes;
+
+import com.google.appengine.api.datastore.Key;
+
+/**
+ * Class representing an individual event.
+ *
+ */
+public final class Cafeteria {
+ 
+  private final String school;
+  private final String name;
+  private final int maxCapacity;
```

Comment:
the var should be `max_capacity` to be consistent with other variables

Reply:
Changed it to max_capacity and meal_time, not sure why I used different naming patterns :(